package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "h5p.components-1.0",
   "version": "1.0.0",
-  "description": "This component library can be used to quickly and easily get a unified look across different content types",
+  "description": "",
+  "main": "index.js",
   "scripts": {
-    "lint": "eslint 'src/**/*.js'",
-    "lint:fix": "eslint --fix 'src/**/*.js'"
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
   },
   "author": "H5P",
   "license": "MIT",
-  "dependencies": {
-    "@babel/eslint-parser": "^7.27.5",
+  "devDependencies": {
     "eslint": "^8.57.1",
     "eslint-config-airbnb-base": "^15.0.0",
```

Comment:
I don't think we need `eslint-config-airbnb-base` and `eslint-plugin-import` since it is part of `eslint-config-h5p`?

Reply:
we need them

Reply:
I see it is listed in their documentation as something we need. I think it _should_ be fine to not install them, but since the documentation says so we'll do that for now. We can take a look at it in another issue.